### ISSUE TITLE
Support quota for core and memory

### ISSUE THREAD

**Issue by ZDWWWWW** [Project-HAMi/HAMi#302]
### 1. Issue or feature description
resourceQuota or volcano-queue can not limit core and memory between different users.
for example:
```go
apiVersion: v1
kind: Pod
metadata:
  name: gpu-pod
spec:
  containers:
    - name: ubuntu-container
      image: ubuntu:18.04
      command: ["bash", "-c", "sleep 86400"]
      resources:
        limits:
          nvidia.com/gpu: 2 # requesting 2 GPUs
          nvidia.com/gpumem: 1000 # Each vGPU contains 1000m device memory （Optional,Integer）
          nvidia.com/gpucores: 10 # Each vGPU uses 10% of the entire GPU （Optional,Integer)
```
Since the usage of gpucores and gpumem is related to the number of GPUs used, for example, gpu:2, gpucores:10 means that the two cards use 10core respectively, and the total usage is 20core.

resourceQuota or volcano-queue can not limit core and memory between different users.
1. Core and mem are not registered on the node, and resourcequota or volcano-queue cannot sense this resource.
2. Since the usage of gpucores and gpumem is related to the usage of gpu, core and mem cannot be deducted correctly.

**Comment by lengrongfu** [comment]
@ZDWWWWW Have you solved this requirement?

**Comment by ZDWWWWW** [comment]
@lengrongfu  yes,resolved by making two changes: one is registering vcore and vmemory to kubelet, and the other is modifying the semantics of vcore and vmemory in the pod specification to represent the total amount of vGPU resources used.
But this isn't a clean solution because vcore and vmemory are just numbers registered, without actually doing anything useful in Kubernetes.

**Comment by lengrongfu** [comment]
@ZDWWWWW How can I contact you? Can you give me your WeChat ID?

Now I want to solve this problem, I need to investigate the user scenario first
